source/index.html.md

```diff
@@ -291,6 +291,7 @@ error categories, the last two digits define specific errors.
 * 1004: Deprecated functionality
 * 1005: Invalid operation
 * 1006: No content for given mobile client name (204 code)
+* 1007: Could not share wire instructions
```

Comment:
As discussed in the engineering channel - I would mark this as `Internal use` and document it somewhere else (TBC)

Reply:
Done -> 42d44d9